src/components/implementation/tests/patina_chan_bench_inter_recv/doc.md

```diff
@@ -0,0 +1,9 @@
+## tests
+
+This is the skeleton interface used by the `mkcomponent.sh` script to aid in the creation of a new component.
```

Comment:
Ditto the issue before. That message applies to all of these doc.md files. Make useful, or remove.